### ISSUE TITLE
XMP?

### ISSUE THREAD

**Issue by mmjaeger** [cnlpete/image-metadata-parser#3]
Hello
You're mentioning XMP in your readme file but I don't see any code in your class referring to XMP parsing.

**Comment by cnlpete** [comment]
YEah, intially i wanted to add xmp parsing aswell, but never got around to it, so this remains a feature request ... sry
